OptimoveSDK/Sources/Classes/Optimobile.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  ViewController.swift
+//  Optimobile
+//
+//  Created by Barak Ben Hur on 04/04/2022.
+//
+
+import UIKit
+import OptimoveCore
+import CoreLocation
+
+class Optimobile {
+    
+    private static let internalQueue = DispatchQueue(label: "com.singletioninternal.queue",
+                                                     qos: .default,
+                                                     attributes: .concurrent)
+    private static let _sheard = Optimobile()
+    
+    public static var sheard: Optimobile {
+        get {
+            return internalQueue.sync {
+                _sheard
+            }
+        }
+    }
+    
+    public typealias deepLinkComplition = (_ deepLinkResolution: DeepLinkResolution) -> ()
```

Comment:
minor spelling note "Complition" -> "Completion"

Reply:
Renaming pass to follow when combining the public APIs